src/components/inputs/SelectInput.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <q-select
+    clearable
+    v-model="localValue"
+    :options="options"
+    :rules="[
+      (value) => isRequired($t, value, attribute.definition.required),
```

Comment:
forgot to add the `definition?.required` ?

Reply:
on purpose, as we discuss yesterday.
no def = no selectInput